pages/news/2020-XX-XX-new-in-2-3-rekordbox-support.html

```diff
@@ -0,0 +1,58 @@
+
+Mixxx 2.3 will support reading the following from Rekordbox prepared removable devices on all of Mixxx's supported platforms:
+
+* Folders
+* Playlists
+* Beatgrids
+* Hotcues
+* Memory cues
+* Loops
+
+You can use this new feature now in [Mixxx 2.3 beta](/download/#unstable). This includes importing the key and BPM analyzed by Rekordbox, and comments and colors for tracks, cues and loops. **We have to be clear however:** this feature *only* reads Rekordbox prepared removable devices, such as USB flash drives or external hard drives. It *will not* read your locally stored Rekordbox collection if you also have Rekordbox installed. The main Rekordbox database (both for Rekordbox 5 and Rekordbox 6) use completely different formats from the USB drives, so reading those databases would be a whole other project.
+
+## Importing from Rekordbox to Mixxx
+
+After you have exported music to a USB drive in Rekordbox, simply plug it in while Mixxx is running. Click on Rekordbox in the library, and all USB drives prepared with Rekordbox will be visible (attach as many as you like!). Then click on the desired removable device, and all your folders and playlist that you have so tirelessly prepared will be revealed for you to play from.
+
+<div style="text-align: center;"><img src="{% static '/static/images/news/mixxx-rekordbox-usb.png' %}" /></div>
+
+Some notes to keep in mind on how Rekordbox hotcues memory cues and loops are imported: Mixxx currently has one main cue point, one loop, and 38 hotcues. As such, all Rekordbox hotcues are mapped to Mixxx hotcues as expected, the first chronological Rekordbox memory cue is mapped to the Mixxx main cue, and the first chronological Rekordbox loop mapped to the Mixxx loop. All loops (including the first loop), and all subsequent Rekordbox memory cues are appended as Mixxx hotcues following the previously imported Rekordbox hotcues. A little confusing? Perhaps, but hopefully it will all make sense when you give it a whirl. Rest assured the hotcue and memory cue colors you assigned in Rekordbox are also imported, assisting in distinguishing between them. Also note that whilst the additional imported Rekordbox loops only appear in Mixxx 2.3 as hotcues, their loop details are preserved, ready for the multiple loop feature proposed for Mixxx 2.4!
+
+<div style="text-align: center;"><img src="{% static '/static/images/news/rekordbox-6-usb.png' %}" /><img src="{% static '/static/images/news/mixxx-rekordbox-usb.png' %}" /></div>
+
+Also note that as with the Serato library feature{# TODO: Add link to serato blogpost #}, we have tried our best to mitigate against certain edge cases of audio files encoded in lossy formats (MP3 and AAC/MP4), where different software's decoders can interpret conflicting timing information, leading to shifted cue points and beatgrids. We hope these are all correct, but some cases may be milliseconds off, and if you find any of these, please let us know on [Zulip](https://mixxx.zulipchat.com).
```

Comment:
Also contains a TODO

Reply:
Serato news post hasn't been published yet so unfortunately don't have a link to update with.